2025-05-10 01:59:52,623 - INFO - Loaded 55 proxies from proxies.txt
2025-05-10 01:59:52,624 - INFO - Starting view bot targeting: https://medium.com/@cgpt/is-vibe-coding-cheating-or-the-future-of-programming-in-the-workplace-216fe9d5084d
2025-05-10 01:59:52,624 - INFO - Sending 50 requests with 5 concurrent workers
2025-05-10 01:59:53,143 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://47.108.159.113:8443
2025-05-10 01:59:53,962 - ERROR - \u274c Error: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))..., proxy=http://41.59.90.175:80
2025-05-10 01:59:56,157 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://182.44.9.76:8072
2025-05-10 01:59:58,218 - INFO - \u2705 Success: status=200, proxy=http://149.129.226.9:1080
2025-05-10 02:00:01,163 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://39.102.214.152:8080
2025-05-10 02:00:02,107 - ERROR - \u274c Error: HTTPSConnectionPool(host='medium.com', port=443): Max retries exceeded with url: /@cgpt/is-vibe-codi..., proxy=http://103.86.109.38:80
2025-05-10 02:00:02,653 - ERROR - \u274c Error: HTTPSConnectionPool(host='medium.com', port=443): Read timed out. (read timeout=10)..., proxy=http://143.0.243.75:8080
2025-05-10 02:00:04,832 - ERROR - \u274c Error: HTTPSConnectionPool(host='medium.com', port=443): Max retries exceeded with url: /@cgpt/is-vibe-codi..., proxy=http://8.213.222.157:3128
2025-05-10 02:00:08,240 - ERROR - \u274c Error: ('Connection aborted.', BadStatusLine('<html>\r\n'))..., proxy=http://50.114.33.143:8080
2025-05-10 02:00:08,761 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://47.104.160.169:443
2025-05-10 02:00:08,858 - ERROR - \u274c Error: HTTPSConnectionPool(host='medium.com', port=443): Max retries exceeded with url: /@cgpt/is-vibe-codi..., proxy=http://89.58.28.110:80
2025-05-10 02:00:09,843 - INFO - \u2705 Success: status=200, proxy=http://8.212.165.164:1000
2025-05-10 02:00:10,227 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://101.200.158.109:8008
2025-05-10 02:00:13,514 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://183.240.46.42:80
2025-05-10 02:00:14,673 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://47.121.129.129:8443
2025-05-10 02:00:17,981 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://39.102.213.3:1111
2025-05-10 02:00:18,529 - INFO - \u2705 Success: status=200, proxy=http://47.91.115.179:8080
2025-05-10 02:00:21,865 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://47.121.183.107:8080
2025-05-10 02:00:22,028 - ERROR - \u274c Error: HTTPSConnectionPool(host='medium.com', port=443): Read timed out. (read timeout=10)..., proxy=http://38.49.129.205:999
2025-05-10 02:00:22,477 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://39.100.88.89:9080
2025-05-10 02:00:23,838 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://47.121.182.36:3128
2025-05-10 02:00:28,519 - ERROR - \u274c Error: HTTPSConnectionPool(host='medium.com', port=443): Max retries exceeded with url: /@cgpt/is-vibe-codi..., proxy=http://159.65.245.255:80
2025-05-10 02:00:30,285 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://8.130.34.237:5555
2025-05-10 02:00:33,045 - ERROR - \u274c Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by t..., proxy=http://8.130.37.235:9080
